android/quest/src/main/assets/configs/app/translations/strings_sw_config.properties

```diff
@@ -0,0 +1,2 @@
+name.title=Bwana.
+gender.female=Kike
```

Comment:
I think we will need `strings_en_config.properties`  for English translation once the `src/main.strings.properties` is deleted from the engine module. I have tested this on the app and it is defaulting to using the properties from the engine module which should not be the case. But adding `strings_en_config.properties` to quest app's translation directory overrides the engine module's translation.

Reply:
The implementation also supports a default with no locale suffix so `strings_config.properties` will suffice